### PR TITLE
Change OG-image according to required size

### LINKED ISSUE #62: Incorrect image size for OG:image meta tag
**Is your feature request related to a problem?**
The current OG:image isn't showing properly when shared.

**Describe the solution you'd like**
1. Make a new image of logo (200px x 200px)
2. Make changes in the code accordingly.

**Screenshot**
Screenshot from 2020-07-17 13-04-42

**Are you working on this issue?**
Yes

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose
The purpose of this PR is to fix #62 

## Goals
To fix the incorrect image size of the current shareable OG:image.

## Approach
1. Added a new image in the correct scale
2. Made changes in the code accordingly.

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/src/index.html [hunk 1]
```diff
@@ -11,8 +11,8 @@
   <!-- Facebook OG tags -->
   <meta property="og:title" content="Academix | Sustainable Education Foundation"/>
   <meta property="og:type" content="website"/>
-  <meta property="og:url " content="https://sefglobal.org/academix/"/>
-  <meta property="og:image" content="https://sefglobal.org/academix/img/academix-logo.png"/>
+  <meta property="og:url" content="https://sefglobal.org/academix/"/>
+  <meta property="og:image" content="https://sefglobal.org/academix/img/academix-og-image.png"/>
```
Comment (jayasanka-sack) [inline app/src/index.html:15]: This should be https://sefglobal.org/academix/public/academix-og-image.png, right?

File: app/src/index.html [hunk 2]
```diff
@@ -11,8 +11,8 @@
   <!-- Facebook OG tags -->
   <meta property="og:title" content="Academix | Sustainable Education Foundation"/>
   <meta property="og:type" content="website"/>
-  <meta property="og:url " content="https://sefglobal.org/academix/"/>
-  <meta property="og:image" content="https://sefglobal.org/academix/img/academix-logo.png"/>
+  <meta property="og:url" content="https://sefglobal.org/academix/"/>
```
Comment (jayasanka-sack) [inline app/src/index.html:14]: I think better to remove OG URL since the current URL is self-descriptive.

Comment (Gimhan-minion) [reply app/src/index.html]: Done

File: app/src/index.html (hunk 1 above)
Comment (Gimhan-minion) [reply app/src/index.html]: Thank for the guidance in this!

**Pull request merged**